smartseq2_single_sample/check_reference/tasks/picard.wdl

```diff
@@ -0,0 +1,56 @@
+task CollectMultipleMetrics {
+  File aligned_bam
+  File ref_genome_fasta
+  String output_filename
+  
+  command {
+    java -Xmx4g -jar /usr/gitc/picard.jar CollectMultipleMetrics \
+      VALIDATION_STRINGENCY=SILENT \
+      METRIC_ACCUMULATION_LEVEL=ALL_READS \
+      INPUT=${aligned_bam} \
+      OUTPUT="${output_filename}" \
+      FILE_EXTENSION = ".txt" \
+      PROGRAM = CollectAlignmentSummaryMetrics \
+      PROGRAM = CollectInsertSizeMetrics \
+      PROGRAM = CollectGcBiasMetrics \
+      PROGRAM = RnaSeqMetrics \
+      REFERENCE_SEQUENCE=${ref_genome_fasta} \
+      ASSUME_SORTED=true
+
+      tar -cvf "${output_file}.tar" ${output_file}.*.txt
+  }
+  output {
+    File alignment_metrics = "${output_filename}.tar"
+  }
+  runtime {
+    docker:"broadinstitute/genomes-in-the-cloud:2.3.1-1504795437"
+    memory:"4 GB"
```

Comment:
saves money to use 3.75gb if you're able to. Otherwise you might as well use 7.5gb. There are a few more cases across the PR where I would look into this.